### PR TITLE
add transcript for episode 058

### PR DESCRIPTION
**Pull request by ericseppanen**
These transcripts are getting a little easier. Either we're putting out better audio, or the AWS transcribe service got better recently. I actually suspect a little of both.

### CODE REVIEW

**Comment by ericseppanen** [comment]
Random anecdote: "ripgrep" appears 8 times in the audio. AWS transcribe tried to spell it 6 (!) different ways: rip grip, rip rep, rip rap, rip Grap, rip grab, rip crap.

File: _transcripts/058-rust-1.58-1.59.md [hunk 1]
```diff
@@ -0,0 +1,1176 @@
...
+include debug info. And in particular, it includes debug info from things like
+the standard library. It includes debug info from your own code. There are ways
+to turn it off of course, and it's been possible to strip that out after the
+fact manually. Like you can use the `strip` command, for example. But now Cargo
+and `rustc` support stripping out this debug info when the binary is linked. You
+might wonder why do you want to do this? Well, the primary driver for this tends
+to be file size. Usually when you end up with very large binaries, it's really
+just because they have a lot of debug info in them. And stripping them can often
+make them significantly larger. Like we're talking order of magnitude here. And
+so if you have a binary that you know you're deploying somewhere where you don't
```
Comment (ericseppanen) [inline _transcripts/058-rust-1.58-1.59.md:837]: Strip can make them "significantly larger." ??? Maybe add *(editor's note: smaller)*

File: _transcripts/058-rust-1.58-1.59.md [hunk 2]
```diff
@@ -0,0 +1,1176 @@
...
+aware, and hopefully we can drive the ecosystem forward in that way.
+
+There's another neat addition in 1.59 that hits probably a small number of
+people, but it's going to make them very happy, which is the ability to create
+stripped binaries. So in general, Rust tends to develop, or build binaries that
+include debug info. And in particular, it includes debug info from things like
+the standard library. It includes debug info from your own code. There are ways
+to turn it off of course, and it's been possible to strip that out after the
+fact manually. Like you can use the `strip` command, for example. But now Cargo
+and `rustc` support stripping out this debug info when the binary is linked. You
```
Comment (ericseppanen) [inline _transcripts/058-rust-1.58-1.59.md:832]: Is "debug info" right? (debug-info, debuginfo?)

File: _transcripts/058-rust-1.58-1.59.md (hunk 1 above)
Comment (jonhoo) [reply _transcripts/058-rust-1.58-1.59.md:837]: Heh, yeah, editor's note seems good here.

File: _transcripts/058-rust-1.58-1.59.md (hunk 2 above)
Comment (jonhoo) [reply _transcripts/058-rust-1.58-1.59.md:832]: https://blog.rust-lang.org/2022/02/24/Rust-1.59.0.html#creating-stripped-binaries uses "debuginfo", so let's use that here too.

**Comment by ericseppanen** [comment]
Ready to go.

**Review by jonhoo — APPROVED** [review]

**Comment by jonhoo** [comment]
Thanks!

**Pull request merged**